src/index.css

```diff
@@ -3,3 +3,11 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+
+.hidden{
```

Comment:
I can't find any usages of the `.hidden` class? If it's not used, it might be better to remove it.

Reply:
Done